google.com.verification.json

```diff
@@ -0,0 +1,20 @@
+{
+  "providerId":"google.com",
+  "providerName":"Google",
+  "serviceId":"verification",
+  "serviceName":"Google GSuite Setup",
```

Comment:
should be:
"serviceName": "Domain Verification",

Reply:
Done